### PR TITLE
Fix over eager assignments 

### PR DESCRIPTION
**Pull request by ankita240796**
## Explanation
Fixes over eager notifications by Oppiabot by adding a separate check to assign project owners.

## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by kevinlee12** [comment]
@ankita240796 Did you run through the checklist?

**Comment by ankita240796** [comment]
Done @kevinlee12, updated the checklist as well.

File: lib/checkPullRequestLabels.js [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+module.exports.checkAssignee = async function(context) {
+  var pullRequest = context.payload.pull_request;
+  var pullRequestNumber = pullRequest.number;
+
+  // eslint-disable-next-line no-console
+  console.log('RUNNING ASSIGNEE CHECK ON PULL REQUEST ' +
+    pullRequestNumber + ' ...');
+
+  var changelogLabel = context.payload.label.name.trim();
+  var hasChangelogLabel = changelogLabel.toUpperCase().startsWith('CHANGELOG');
```
Comment (kevinlee12) [inline lib/checkPullRequestLabels.js]: Just to verify, this won't trigger if Oppiabot puts a PR merge conflict label right?

Comment (ankita240796) [reply lib/checkPullRequestLabels.js]: No, it will only trigger if the label name starts with CHANGELOG, which is not the case for merge conflicts label.

**Pull request merged**